src/main/java/TaskType.java

```diff
@@ -0,0 +1,3 @@
+public enum TaskType {

```

Comment:
Good call:) Easier to manage task types.

Reply:
Thanks!